embeddings/config/lightning_config.py

```diff
@@ -125,6 +125,9 @@ class LightningQABasicConfig(LightningBasicConfig):
     )
     doc_stride: int = 64
     max_seq_length: int = 128
+    # those type: ignores are necessary for mypy to pass
```

Comment:
I do not agree. Mypy is correct because it expects `Dict[str,Any]` and you pass `None`. Change type for `Optional[Dict[str,Any]` e.g.
```python
task_model_kwargs: Optional[Dict[str, Any]] = None
```

Reply:
If we do for example `task_model_kwargs: Optional[Dict[str, Any]] = None`, mypy throws another type of error: ` error: Incompatible types in assignment (expression has type "Optional[Dict[str, Any]]", base class "LightningBasicConfig" defined the type as "Dict[str, Any]")`.  I think the solution might be:
`task_model_kwargs: Optional[Dict[str, Any]] = None. # type: ignore`. More accurate, but still with `type: ignore`

Reply:
Ahh ok, I will think about other solution.